src/views/Applications/Application.vue

```diff
@@ -1,9 +1,109 @@
 <template>
-  <div></div>
+  <div class="container py-4">
+    <div class="row">
+      <div class="col">
+        <a-card>
+          <div class="d-flex">
+            <img :src="applicationIcon" class="card__icon">
+            <div class="card__title d-flex flex-column">
+              <span><strong>{{applicationTitle}}</strong></span>
+              <span><small>Categoria</small> {{applicationCategory}}</span>
+              <span><small>Por</small> {{applicationAuthor}}</span>
+              <span>
+                <a-button type="primary" size="small" @click="installApp">Instalar Agora</a-button>
+              </span>
+            </div>
+          </div>
+        </a-card>
+      </div>
+    </div>
+    <div class="row py-4">
+      <div class="col">
+        <a-card :loading="loading">
+          <vue-markdown>{{applicationDescription}}</vue-markdown>
+        </a-card>
+      </div>
+    </div>
+  </div>
 </template>
-
 <script>
```

Comment:
Por favor deixe pelo menos o script em arquivo separado, facilita os linters e reutilização, eg.:

```suggestion
<script src="./js/Application.js"></script>
```